libs/platform/src/lib/value-help-dialog/components/select-tab/select-tab.component.html

```diff
@@ -37,7 +37,7 @@ <h5 fd-title role="heading" aria-level="5">
                     <fd-checkbox
                         *ngIf="isMultiSelection"
                         [name]="uid + '-row-all'"
-                        [compact]="true"
+                        [compact]="contentTableDensity === _contentDensityOptions.COMPACT"
```

Comment:
`!== 'cozy'` should work?

Reply:
Seems like it's still actual. Pipe should be used.

Reply:
yes, thanks. fixed